src/effects.cpp

```diff
@@ -276,7 +276,7 @@ void LoadEffectFactories()
         ADD_EFFECT(EFFECT_MATRIX_SMPICASSO3IN1,     PatternSMPicasso3in1, "Lines", 38);
         ADD_EFFECT(EFFECT_MATRIX_SMPICASSO3IN1,     PatternSMPicasso3in1, "Circles", 73);
         ADD_EFFECT(EFFECT_MATRIX_SMAMBERRAIN,       PatternSMAmberRain);
-        ADD_EFFECT(EFFECT_MATRIX_SMSTROBE_DIFFUSION,PatternSMStrobeDiffusion);
+ //     ADD_EFFECT(EFFECT_MATRIX_SMSTROBE_DIFFUSION,PatternSMStrobeDiffusion);
         ADD_EFFECT(EFFECT_MATRIX_SMRAINBOW_TUNNEL,  PatternSMRainbowTunnel);

```

Comment:
I'm sure I had a good reason for turning off the Diffusion effect, but I think I just didn't like it.  If you or anyone else is partial to it, let me know.

Reply:
To be honest, if it's just a matter of you not liking it then I'd like to suggest leaving it in and allowing the user to disable it with the web UI. That's what that feature is for.

If there is another good reason for disabling it, then that's fine of course. In which case I do (indeed) have a mild curiosity what the problem with it is.

Reply:
Ok, keeping it